cmd/bbolt/command_stats.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+func newStatsCommand() *cobra.Command {
+	statsCmd := &cobra.Command{
+		Use:   "stats <bbolt-file>",
+		Short: "print stats of bbolt database",
+		Long: strings.TrimLeft(`
+usage: bolt stats PATH
+
+Stats performs an extensive search of the database to track every page
+reference. It starts at the current meta page and recursively iterates
+through every accessible bucket.
+
+The following errors can be reported:
+
+    already freed
+        The page is referenced more than once in the freelist.
+
+    unreachable unfreed
+        The page is not referenced by a bucket or in the freelist.
+
+    reachable freed
+        The page is referenced by a bucket but is also in the freelist.
+
+    out of bounds
+        A page is referenced that is above the high water mark.
+
+    multiple references
+        A page is referenced by more than one other page.
+
+    invalid type
+        The page type is not "meta", "leaf", "branch", or "freelist".
+
+No errors should occur in your database. However, if for some reason you
+experience corruption, please submit a ticket to the etcd-io/bbolt project page:
+
+  https://github.com/etcd-io/bbolt/issues
+`, "\n"),
+		Args: cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			prefix := ""
+			if len(args) > 1 {
+				prefix = args[1]
+			}
```

Comment:
The `prefix` should be option instead of an argument. But keeping it as an argument also keeps the behaviour, so it doesn't break the user experience. I am OK either way.

cc @fuweid @ivanvc @tjungblu

Reply:
thanks for your review, yes i tried to minimize the changes as much as I can, for this purpose